TravelBug/src/WebPages/loginPage/LoginPage.jsx

```diff
@@ -0,0 +1,102 @@
+import "./Loginpage.css";
+import { useState } from "react";
+import { useAuth } from "../../Contexts/AuthContext";
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { useData } from "../../Contexts/Datacontext";
+
+const LoginPage = () => {
+  const navigate = useNavigate();
+  const { loginCall, token, user } = useAuth();
+  const { dispatch } = useData();
+  const [loginDetails, setLoginDetails] = useState({ email: "", password: "" });
+
+  const guestLoginHandler = () => {
+    setLoginDetails({
+      email: "adarshbalika@gmail.com",
+      password: "adarshbalika",
+    });
+    loginCall("adarshbalika@gmail.com", "adarshbalika");
+  };
+
+  useEffect(() => {
+    if (token) {
+      setTimeout(() => {
+        navigate("/");
+      }, 1000);
+    }
+  }, [token]);
+
+  return (
+    <>
+      <div className="login-page">
+        <div className="login-container login-box">
+          <div className="login-inputs">
+            <h1>Login</h1>
+            <div className="input">
+              <label htmlFor="login-email">Email</label>
+              <input
+                id="login-email"
+                value={loginDetails.email}
+                className="input-txt"
+                type="email"
+                onChange={(e) =>
+                  setLoginDetails({ ...loginDetails, email: e.target.value })
+                }
+              />
+            </div>
+            <div className="input">
+              <label htmlFor="login-password">Password</label>
+              <input
+                id="login-password"
+                value={loginDetails.password}
+                className="input-txt"
+                type="password"
+                onChange={(e) =>
+                  setLoginDetails({ ...loginDetails, password: e.target.value })
+                }
+              />
+            </div>
+
+            <div className="input htmlFot-password">
```

Comment:
I think you meant to include the htmlFor attribute here, it has gotten included with the className.